scriptmodules/supplementary/emulationstation/configscripts/retroarch.sh

```diff
@@ -319,6 +309,26 @@ function map_retroarch_keyboard() {
 function onend_retroarch_joystick() {
     local device_type=$1
     local device_name=$2
+    
+    # hotkey sanity check
+    # remove hotkeys if there is no hotkey enable button
+    if ! grep -q "input_enable_hotkey" /tmp/tempconfig.cfg; then
+        iniSet "input_state_slot_decrease_btn" ""
+        iniSet "input_state_slot_increase_btn" ""
+        iniSet "input_reset_btn" ""
+        iniSet "input_menu_toggle_btn" ""
+        iniSet "input_load_state_btn" ""
+        iniSet "input_save_state_btn" ""
+        iniSet "input_exit_emulator_btn" ""
+        iniSet "input_state_slot_decrease_axis" ""
+        iniSet "input_state_slot_increase_axis" ""
+        iniSet "input_reset_axis" ""
+        iniSet "input_menu_toggle_axis" ""
+        iniSet "input_load_state_axis" ""
+        iniSet "input_save_state_axis" ""
+        iniSet "input_exit_emulator_axis" ""
+    fi
```

Comment:
Sorry to comment on an ancient PR - but I had a question about the above block (I am making some changes to how hotkey configuration works).

Why do these need to be set to empty if the user has no hotkey enable (because they have no select button) ? Does retroarch enable them by default ? I would have thought if they didn't exist in the controller file, they are not enabled. I did a test and I couldn't see any issues either.